crates/cxx-qt-gen/test_inputs/invokables.rs

```diff
@@ -52,12 +52,14 @@ mod ffi {
 
     impl cxx_qt::Threading for MyObject {}
 
-    impl
+    impl<'a>
         cxx_qt::Constructor<
-            (i32, *mut QObject),
+            (i32, &'a mut QString),
             BaseArguments = (*mut QObject,),
-            NewArguments = (i32,),
+            NewArguments = (&'a QString,),
```

Comment:
any reason this isn't `mut` ? or the other isn't const ?

Reply:
I just wanted to show off the `default` for these two types.
E.g. the `*mut QObject` should map to a `QObject*` (usually the parent pointer). The `&'a QString` should map to a `const QString &`, which is usually what you would use in C++.

Reply:
But in C++ you get `QString&` for the constructor as the template type as the top is `&mut QString` ?

Reply:
(at least looking at invokables.h in the test outputs)

Reply:
Aaah, that `mut` you mean.
Good point.
Fixed.